package.json

```diff
@@ -37,7 +38,8 @@
     "@fortawesome/free-regular-svg-icons": "5.15.4",
     "@fortawesome/free-solid-svg-icons": "5.15.4",
     "@fortawesome/react-fontawesome": "0.2.0",
-    "algoliasearch": "4.17.2",
+    "@pact-foundation/pact": "^11.0.2",
+    "algoliasearch": "4.18.0",
```

Comment:
Is this package upgrade required?

Reply:
I don't think so, it might have automatically updated my algoliasearch package when tinkering with the dependencies. I'll set it to 4.17.2 and see how it runs.

Reply:
In terms of keeping commits/PRs focused on a particular set of changes, I'd revert this line (which I think is what Tim's implying) and the associated changes in package-lock.json.  

The easiest way to do that is just edit it back to 4.17.2 in this file manually, then run `npm i` to get the package-lock.json correct.  Editing package-lock.json manually is never a good idea!